README.md

```diff
@@ -329,6 +329,57 @@ Returns a uniformly random word of length `k`
 dfa.random_word(1000, seed=42)
 ```
 
+#### DFA.predecessor(self, input_str, strict=True, key=None)
+
+Returns the first string accepted by the DFA that comes before
+the input string in lexicographical order.
+See `DFA.successors` for more information.
+
+```python
+prev_word = dfa.predecessor('0110')
+same_word = dfa.predecessor(prev_word, strict=False)
+```
+
+#### DFA.predecessors(self, input_str, strict=True, key=None)
+
+Generates all strings that come before the input string
+in lexicographical order.
+See `DFA.successors` for more information.
+
+```python
+# Generates all strings in a language in lexographical order
+for word in dfa.predecessors(None):
+    print(word)
+```
+
+#### DFA.successor(self, input_str, strict=True, key=None)
+
+Returns the first string accepted by the DFA that comes after
+the input string in lexicographical order.
+See `DFA.successors` for more information.
+
+```python
+next_word = dfa.successor('0110')
+same_word = dfa.predecessor(next_word, strict=False)
+```
+
+#### DFA.successors(self, input_str, strict=True, key=None, reverse=False)
+
+Generates all strings that come after the input string
+in lexicographical order.
+Passing in `None` will generate all words.
+If `strict` is set to `False` and `input_str` is accepted by the DFA then
```

Comment:
@Tagl What do you think about inverting the associated logic and renaming this flag to `inclusive` (default: `True`)?

But of course, if `strict` would make more sense to people who understand the underlying theory of this operation, then we can keep as `strict`.

Reply:
Normally when people talk of successors and predecessors they implicitly mean strict successors and strict predecessors.
Originnally I had this as `include_input` but I feel like `strict` is more appropriate. The default behavior should be to skip the input string since successors usually means strict successors.

Reply:
@Tagl Makes sense to me! Then we shall keep the parameter name as `strict`. 🙂